### PR TITLE
Update fake CEGA to v0.2.0

### PR DESCRIPTION
**Pull request by jbygdell**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [X] New feature
- [X] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Changes made:

1) Updates MQ to 3.7.8 and add TLS option.
2) Add TLS to cega-users and match the version of the server script with latest LocalEGA
3) Delete obsolete files.
### Related issues:

#32 #38

### CODE REVIEW

**Comment by jbygdell** [comment]
#38 and #39 has a bit of round robin issues, As the tests will not pass until both are applied and the e2e tester is up to date.

File: ega-charts/cega/values.yaml [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-  imagePullPolicy: Always
-
 persistence: 
   enabled: true
 
 secrets:
-  cega_users_pass: 
+  cega_users_pass: ""
+
+tls: true
```
Comment (blankdots) [inline ega-charts/cega/values.yaml:10]: missing indentation ?

**Review by blankdots — APPROVED** [review]
small comment but otherwise approved.

File: ega-charts/cega/values.yaml (hunk 1 above)
Comment (jbygdell) [reply ega-charts/cega/values.yaml:10]: No it's a global var for all of fake cega.

Comment (blankdots) [reply ega-charts/cega/values.yaml:10]: ok.

**Comment by blankdots** [comment]
Closing in favour of #41

**Pull request closed without merging**